pkg/cardinality/request.go

```diff
@@ -264,3 +264,62 @@ func extractCountMethod(values url.Values) (countMethod CountMethod, err error)
 		return "", fmt.Errorf("invalid 'count_method' param '%v'. valid options are: [%s]", countMethodParams[0], strings.Join([]string{string(ActiveMethod), string(InMemoryMethod)}, ","))
 	}
 }
+
+type ActiveSeriesRequest struct {
+	MatcherSet [][]*labels.Matcher
```

Comment:
Yeah, I wasn't sure either, but I thought there's maybe a good reason why prometheus accepts multiple selectors and it can't hurt to be compatible in that case.

Happy to make it single value only though if we don't think we'd ever need more than one selector.

Reply:
That sounds sane to me. I have changed the logic to accept a single `selector` parameter.